lua/terrortown/entities/items/item_ttt_armor.lua

```diff
@@ -9,7 +9,7 @@ ITEM.EquipMenuData = {
 	desc = "item_armor_desc"
 }
 ITEM.material = "vgui/ttt/icon_armor"
-ITEM.CanBuy = {ROLE_TRAITOR, ROLE_DETECTIVE}
+ITEM.CanBuy = nil
```

Comment:
why?

Reply:
Because I wanted to keep the old item as a fallback but it should no be buyable. Might add the new functionality to the old item, but it has to be rebuyable